### PR TITLE
swift-format 0.50600.0

### PR DESCRIPTION
**Pull request by scottjanderson**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by scottjanderson** [comment]
I wasn't expecting the `brew bump-formula-pr` tool to automatically submit the PR – should have read the docs!

Might need to also bump the Xcode version to 13.3 to match what 'swift-format' expects:
https://github.com/apple/swift-format/tree/0.50600.0#matching-swift-format-to-your-swift-version

That will likely mean we should also remove Big Sur, as Xcode 13.3 requires a Mac running macOS Monterey 12 or later:
https://developer.apple.com/documentation/xcode-release-notes/xcode-13_3-release-notes

Happy to do that in this PR, or in a separate PR if that's preferable.

File: Formula/swift-format.rb [hunk 1]
```diff
@@ -15,17 +15,15 @@ class SwiftFormat < Formula
 
   bottle do
     sha256 cellar: :any_skip_relocation, arm64_monterey: "53959426e357b8be149cd84141c22df630a6d0c8e5097d7353cce3cdfb3738f6"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "64fe92f2761fd21ec2377ec8cb51f9850ec9d6241390d719936bad05ac937d2e"
     sha256 cellar: :any_skip_relocation, monterey:       "3b5dc99305ebf489c4f465bb9df393a9c9a9de40814288c4b12110a02b35d6c3"
-    sha256 cellar: :any_skip_relocation, big_sur:        "61d4b88004916e7d4456bef630017eebf0636c33cb1086958d200505c1d2a369"
     sha256                               x86_64_linux:   "77aebfbdb7f2c58e8de6cbb2c6d33f534a75c6e268d98e5aa653a388ac1ca8d8"
   end
```
Comment (carlocab) [inline Formula/swift-format.rb:22]: Please don't touch the `bottle` block.

**Comment by Bo98** [comment]
This will require updating our CI to Xcode 13.3 which I will do this coming week.

**Comment by carlocab** [comment]
Does this also need #96853?

**Comment by Bo98** [comment]
Yeah. I should be able to finish that soon now that Go 1.18 is off the TODO list.

**Comment by Bo98** [comment]
Merged this at the same time as https://github.com/Homebrew/homebrew-core/pull/96853. Thanks!

064878f5435dc88cd3c55b6bde3c8647a9b91aa7

**Pull request closed without merging**